api/src/main/java/io/grpc/InternalMetadata.java

```diff
@@ -82,4 +82,19 @@ public static Metadata newMetadata(int usedNames, byte[]... binaryValues) {
   public static int headerCount(Metadata md) {
     return md.headerCount();
   }
+
+  @Internal
+  public static Object[] serializePartial(Metadata md) {
+    return md.serializePartial();
+  }
+
+  @Internal
+  public static <T> Metadata.LazyValue<T> parsedValue(Key<T> key, T value) {
```

Comment:
I guess we should make this return code be `Object`, since LazyValue is hidden? Although we don't really care about ABI here, it also means ABI won't break if the internals change.

Reply:
Done.